lib/yt/collections/reports.rb

```diff
@@ -63,31 +63,39 @@ def within(days_range, country, state, dimension, videos, try_again = true)
         @country = country
         @state = state
         @videos = videos
+
```

Comment:
This is not our code base so I think we should not change code style like adding empty lines and adding padding to parentheses. Make only the required changes, and it will be easier to maintain our fork with less conflicts over time